library/Fission/Web/IPFS.hs

```diff
@@ -38,10 +39,12 @@ type AuthedAPI = Auth :> UnauthedAPI
 type UnauthedAPI = "cids" :> CID.API
               :<|> Upload.API
               :<|> Pin.API
+              :<|> ("dag" :> Dag.API)
 
 type SimpleAPI = "cids" :> CID.API
             :<|> Upload.Simple.API
             :<|> Pin.API
+            :<|> ("dag" :> Dag.API)
```

Comment:
I really should rearrange the order in these types to make them easier to maintain 🤔 In either case, I'm pretty sure that we don't need the parentheses [UPDATE: removed locally and they're not needed)

Reply:
While we're in here:

```haskell
type UnAuthedAPI = CommonAPI :<|> Upload.API
type SimpleAPI   = CommonAPI :<|> Upload.Simple.API

type CommonAPI = "cids" :> CID.API
            :<|> "dag"  :> Dag.API
            :<|> Pin.API
```

This will break order in a few places, but is cleaner. Thoughts?

Reply:
Got labelled as outdated, but isn't :octocat: